plugins/source/azure/resources/services/compute/virtual_machine_assess_patches.go

```diff
@@ -0,0 +1,72 @@
+package compute
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v4"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func virtualMachineAssessPatches() *schema.Table {
+	tableName := `azure_compute_virtual_machine_assess_patches`
+	return &schema.Table{
+		Name:     tableName,
+		Resolver: fetchVirtualMachineAssessPatches,
+		Description: `https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/assess-patches?tabs=HTTP#virtualmachineassesspatchesresult.
+
+This will begin patch assessments on available virtual machines and can take long to complete.
```

Comment:
Took a few (5?) seconds for a single VM, but can delay things a lot since it's done separately for each resource.

Reply:
Yeah, so maybe we should add a commit message override similar to https://github.com/cloudquery/cloudquery/pull/8726 (see `BEGIN_COMMIT_OVERRIDE`)